rocksdb/rocksdb.nim

```diff
@@ -465,6 +465,22 @@ proc releaseSnapshot*(db: RocksDbRef, snapshot: SnapshotRef) =
     rocksdb_release_snapshot(db.cPtr, snapshot.cPtr)
     snapshot.setClosed()
 
+proc flush*(db: RocksDbRef, cfs: openArray[ColFamilyHandleRef]): RocksDBResult[void] =
+  withLock(db.lock):
+    if not db.isClosed():
+      var cfs = cfs.mapIt(it.cPtr)
+      var errors: cstring
+      var opts = rocksdb_flushoptions_create()
```

Comment:
How often do we plan to call flush? If it gets called frequently then perhaps we should create the flush options when starting the database and hold a pointer to it and we can free when closing the database to reduce the number of allocations for performance reasons.